src/shared-components/forms/style.js

```diff
@@ -0,0 +1,99 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import { style as TYPOGRAPHY_STYLE } from '../typography';
+import { COLORS } from '../../constants';
+
+export const FormContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 16px;
+`;
+
+export const errorIconStyles = css`
+  position: relative;
+
+  svg {
+    display: block;
+    width: 16px;
+    height: 16px;
+    position: absolute;
+    top: 50%;
+    margin-top: -8px;
+    right: 16px;
+    fill: ${COLORS.statusRed};
+  }
+`;
+
+export const InputContainer = styled.div`
+  svg {
+    display: none;
+  }
+
+  ${({ shouldRenderError }) => shouldRenderError && errorIconStyles};
+`;
+
+export const baseInputStyles = css`
+  appearance: none;
+  border: 1px solid ${COLORS.border};
+  border-radius: 0;
+  color: ${COLORS.primary};
+  font-size: 16px;
+  padding: 13px 16px;
+  transition: border-color 350ms;
+  width: 100%;
+  outline: none;
+
+  &::placeholder {
+    color: ${COLORS.primaryTint3};
+  }
+`;
+
+export const inputStyles = (showError, focusStyle) => {
+  const borderColor = showError ? COLORS.statusRed : false;
+  const focusBorderColor =
+    focusStyle === 'focusSecondary' ? COLORS.secondary : COLORS.primary;
```

Comment:
focus style should be `borderColor: COLORS.primary` and should include a box shadow under the `focusSecondary` constant

Reply:
@benkolde can you review the focus styles. I added the box-shadow:focusSecondary but it looks very bright noticeable (not subtle?)

Reply:
should be here https://curology-radiance-pr-82.herokuapp.com